bin/oxfordshire/send-rdi-emails

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env perl
```

Comment:
Could this code be factored out into a package in perllib/FixMyStreet/scripts? Would (maybe) make it easier to test too? Certainly would make it easier to reuse if we need to further down the line